charts/devlake/values.yaml

```diff
@@ -17,7 +17,12 @@
 
 # replica count for dev
 replicaCount: 1
-imageTag: v0.17.0-beta10
+imageTag: v0.18.0-beta1
+
+#the common environments for all pods
+envs:
+#  - name: TZ
```

Comment:
IMO, TZ is mandatory.

Reply:
I concur.

Reply:
Thanks for the review!  which TZ to be set, do you have any idea? @matrixji @klesh

Reply:
ref: https://github.com/apache/incubator-devlake-helm-chart/pull/113#discussion_r1169714489